test/ctia/entity/attack_pattern/core_test.clj

```diff
@@ -0,0 +1,91 @@
+(ns ctia.entity.attack-pattern.core-test
+  (:require [clojure.test :refer [deftest is testing join-fixtures use-fixtures]]
+            [ctia.entity.attack-pattern.core :as sut]
+            [ctia.test-helpers.auth :refer [all-capabilities]]
+            [ctia.test-helpers.core :as helpers]
+            [ctia.test-helpers.fake-whoami-service :as whoami-helpers]
+            [ctia.test-helpers.fixtures :as fixtures]
+            [java-time :refer [adjust days instant plus to-java-date]]
```

Comment:
fwiw I scrolled up to check from where the time function came from.
I would use this alias:
```suggestion
            [java-time :as jt]
```